database/src/pool.rs

```diff
@@ -294,3 +294,92 @@ impl Pool {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use chrono::Utc;
+    use std::str::FromStr;
+
+    use super::*;
+    use crate::{tests::run_db_test, Commit, CommitType, Date};
+
+    /// Create a Commit
+    fn create_commit(commit_sha: &str, time: chrono::DateTime<Utc>, r#type: CommitType) -> Commit {
```

Comment:
I've copied this function from another set of tests from `load.rs`. Should we have something like a `test_utils.rs`